core/src/tests/CMakeLists.txt

```
@@ -264,14 +266,29 @@ bareos_add_test(
   COMPILE_DEFINITIONS TEST_TEMP_DIR=\"${TEST_TEMP_DIR}\"
                       TEST_ORIGINAL_FILE_DIR=\"${TEST_ORIGINAL_FILE_DIR}\"
 )
-set_tests_properties(
-  gtest:recent_job_results_list.read_job_results_from_file PROPERTIES LABELS
-                                                                      "broken"
-)
-set_tests_properties(
-  gtest:recent_job_results_list.write_job_results_to_file PROPERTIES LABELS
-                                                                     "broken"
-)
+
+if(${CMAKE_HOST_SYSTEM_PROCESSOR} STREQUAL "sparc")
```

it doesn't make sense to check for the platform we're building on, we should check the platform we're building for.
```suggestion
if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "sparc")
```
Nevertheless, it should be rather simple to provide test-data for big endian, too. So there's no real point in disabling it.

---

if(HAVE_BIG_ENDIAN) ..